charts/enterprise/authelia/values.yaml

```
@@ -228,10 +228,10 @@ authentication_backend:
     ## otherwise it would break the configuration for that user. Technically, non-unique attributes like 'mail' can also
     ## be used but we don't recommend using them, we instead advise to use the attributes mentioned above
     ## (sAMAccountName and uid) to follow https://www.ietf.org/rfc/rfc2307.txt.
-    username_attribute: ""
+    username_attribute: "uid"
 
     ## An additional dn to define the scope to all users.
-    additional_users_dn: OU=Users
+    additional_users_dn: OU=People
```

```suggestion
    additional_users_dn: OU=Users
```

This is only an lldap thing, not native openldap.